docs/mkdocs/docs/features/binary_formats/messagepack.md

```diff
@@ -64,7 +67,8 @@ The library uses the following mapping from JSON values types to MessagePack typ
 
 !!! info "NaN/infinity handling"
 
-	If NaN or Infinity are stored inside a JSON number, they are serialized properly. function which serializes NaN or Infinity to `null`.
+	If NaN or Infinity are stored inside a JSON number, they are serialized properly. function which serializes NaN or
```

Comment:
Tab to 4 spaces. The rest of the sentence is broken. What is that trying to say?
```suggestion
    If NaN or Infinity are stored inside a JSON number, they are serialized properly. function which serializes NaN or
```